### PR TITLE
Issue/2518 rpm openssl version

### PR DESCRIPTION
**Pull request by sanderr**
# Description

part of #2518

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [ ] Changelog entry
- [ ] Type annotations are present
- [ ] Code is clear and sufficiently documented
- [ ] No (preventable) type errors (check using make mypy or make mypy-diff)
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

# Reviewer Checklist:

- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Code is clear and sufficiently documented
- [ ] Correct, in line with design

### CODE REVIEW

File: inmanta.spec [hunk 1]
```diff
@@ -33,15 +33,21 @@ BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
...
-BuildRequires:  openssl-devel
 
 Requires:       git
 Requires:       gcc
-Requires:       openssl
 Requires:       logrotate
 Requires:       libffi
 Requires(pre):  shadow-utils
 
+%if 0%{?el7}
```
Comment (bartv) [inline inmanta.spec:43]: why not: `if 0%{?rhel} == 7`?

Comment (bartv) [reply inmanta.spec:43]: not sure if that works

Comment (sanderr) [reply inmanta.spec:43]: Same thing. I could change it if you want?

Comment (bartv) [reply inmanta.spec:43]: does not really matter I guess.

**Review by bartv — APPROVED** [review]

**Review by arnaudsjs — APPROVED** [review]

**Pull request merged**